### ISSUE TITLE
Path replace is broken

### ISSUE THREAD

**Issue by iphydf** [ZedThree/clang-tidy-review#20]
```
Replacing '/home/runner/work/c-toxcore/c-toxcor' with '/github/workspace'
...
LLVM ERROR: Cannot chdir into "/github/workspacee"!
```

https://github.com/TokTok/c-toxcore/runs/4472823526?check_suite_focus=true#step:6:165

This is our config file: https://github.com/TokTok/c-toxcore/blob/cf9a0b13418ef78cff6ac5c3d8dd1830360bab77/.github/workflows/clang-tidy-review.yml

**Comment by kgfoundrydig** [comment]
Yep, still broken:
clang-tidy failed with return code 1 and error:
LLVM ERROR: Cannot chdir into "/github/workspacet"!

My path ends with a t... it's getting included at the end of the local workspace directory.

Can we use an earlier version to avoid this?

**Comment by ZedThree** [comment]
Fixed by #21

**Comment by ZedThree** [comment]
In release 0.7.1
